.github/workflows/build-manual.yml

```diff
@@ -0,0 +1,65 @@
+name: Manual Build and Test
+
+on:
+  workflow_dispatch:
+    inputs:
+      os:
+        description: 'OS'
+        required: false
+        default: 'ubuntu-latest'
+        type: choice
+        options:
+          - ubuntu-latest
+          - windows-latest
+      jdk-distribution:
+        description: 'JDK Distribution'
+        required: false
+        default: 'temurin'
+        type: choice
+        options:
+          - temurin
+          - semeru
+          - microsoft
+          - oracle
+          - zulu
+          - corretto
+          - liberica
+      jdk-version:
+        description: 'JDK Version'
+        required: false
+        default: '11'
+        type: choice
+        options:
+          - 11
+          - 17
+          - 18
+      args:
+        description: 'Arguments like -Dquickly | -DskipTests | -DallTests'
+        required: false
+        default: '-Dquickly'
+        type: string
+
+jobs:
+  build:
+    name: Build and Test
+    runs-on: ${{ inputs.os }}
+    timeout-minutes: 120
+    strategy:
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK ${{ inputs.jdk-distribution }} ${{ inputs.jdk-version }}
+        uses: actions/setup-java@v3
+        with:
+          distribution: ${{ inputs.jdk-distribution }}
+          java-version: ${{ inputs.jdk-version }}
+          cache: 'maven'
+      - name: Clear any possible SNAPSHOT dependencies in the local maven repository
+        run: |
+          LOCAL_REPO="$HOME/.m2/repository"
+          echo "Cleaning SNAPSHOTS from $LOCAL_REPO"
+          if [ -d "$LOCAL_REPO" ]; then find "$LOCAL_REPO" -name "*SNAPSHOT*" | xargs -I {} rm -rfv "{}"; fi
```

Comment:
Do we need this in this case?

Reply:
It was copied from `build-only.yml`. Not sure if it's really needed in either case. I guess it's more of a just-in-case safety measure.

Reply:
Yes. The reason that was there was to ensure WildFly builds in a clean environment. There was an issue with a profile not correctly building if the server wasn't built first, so the other workflow just verifies that. I don't think we need it here though.